salt/states/boto_ec2.py

```diff
@@ -169,6 +169,8 @@ def eni_present(
         description=None,
         groups=None,
         source_dest_check=True,
+        allocate_eip=False,
```

Comment:
We generally prefer to add new kwargs to the end of the list, since python allows kwargs to be passed positionally.